internal/model/board_test.go

```diff
@@ -729,6 +761,84 @@ func TestBoard(t *testing.T) {
 		})
 	})
 
+	t.Run("getAccessibleNeighbors()", func(t *testing.T) {
+		t.Run("Should return all accessible neighbors for a tile", func(t *testing.T) {
+			board := NewTestBoard()
+			{
```

Comment:
Same, I think you could summarize that to a set of test data. 

- PHPUnit has dataProviders https://docs.phpunit.de/en/10.0/writing-tests-for-phpunit.html#data-providers
- Jest has `test.each` https://jestjs.io/docs/api#testeachtablename-fn-timeout